clean/new_orleans_da_cprr_2021_brady.py

```diff
@@ -0,0 +1,347 @@
+        .str.replace('14:133 filing or maintaining false public records',
+                     '14:133 relative to filing or maintaining false public records', regex=False)\
+        .str.replace('32:51 no vehicle license', '32:51 relative to no vehicle license', regex=False)\
+        .str.replace('32:61 maximum speed limit', '32:61 relative to maximum speed limit', regex=False)\
+        .str.replace('32:50 careless operation of a motor vehicle',
+                     '32:50 relative to careless operation of a motor vehicle', regex=False)\
+        .str.replace('14:138 public payroll fraud', '14:138 relative to public payroll fraud', regex=False)\
+        .str.replace('14:38 simple assault and r.s. 14:63 criminal trespassing',
+                     '14:38 relative to simple assault | r.s. 14:63 criminal trespassing', regex=False)\
+        .str.replace('relative sexual battery', 'r.s. 14:43.1 relative to sexual battery')\
+        .str.replace("to la. r.s. 32:52 relative to driver's license expired",
+                     "to wit: la. r.s. 32:52 relative to driver's license expired", regex=False)\
+        .str.replace(r'^relative to theft$', 'r.s. 14:67 relative to theft')\
+        .str.replace('relative to aggravated incest', 'r.s. 14:78.1 relative to aggravated incest', regex=False)\
+        .str.replace('relative to attempt and conspiracy to commit offense',
+                     '14:26 relative to attempt and conspiracy to commit offense', regex=False)\
+        .str.replace('relative to conspiracy to commit offense',
+                     '14:26 relative to attempt and conspiracy to commit offense', regex=False)\
+        .str.replace('relative to forcible rape', '14:42.1 relative to forcible rape', regex=False)\
+        .str.replace('relative to no proof of insurance',
+                     'r.s. 32:863.1 relative to no proof of liability insurance', regex=False)\
+        .str.replace(r'^relative to simple battery$', 'r.s. 14:35 relative to simple battery')\
+        .str.replace(r'^relative to simple assault$', 'r.s. 14:38 relative to simple assault')\
+        .str.replace(r'^simple assault$', 'r.s. 14:38 relative to simple assault')\
+        .str.replace('r.s. 14:99 relative to reckless operation of a vehicle',
+                     'r.s. 14:99 relative to reckless operation of a motor vehicle', regex=False)\
+        .str.replace('recklesss', 'reckless', regex=False)\
+        .str.replace("r.s. 32:412 driver's must be licensed", "r.s. 32:412 drivers must be licensed", regex=False)\
+        .str.replace('(see attached criminal charges)', '', regex=False)
+    return df.drop(columns=['directive'])
+
+
+def drop_rows_without_last_name(df):
+    df = df[df.last_name != 'test']
+    return df.dropna(subset=['last_name']).reset_index(drop=True)
+
+
+def clean():
+    df = pd.read_csv(data_file_path(
+        'new_orleans_da/new_orleans_da_cprr_2021.csv'))
```

Comment:
The file newly added in download_input is `new_orleans_da/new_orleans_da_brady_list_2021.csv`. The name is wrong.

Reply:
Resolved.